### ISSUE TITLE
Mechanism to populate treetracker.tree from treetracker.capture 

### ISSUE THREAD

**Issue by sebastiangaertner** [Greenstand/treetracker-admin-client#934]
Since only some projects have used a code coming from the App within the treetracker.public.trees.note field to identify the first appearance of a tree other orgs and users need to be able to select captures and turn them into tree records. (capture->tree) in order to follow the new domain model and activate the capture matching tool etc. 
This would need a small UI but could/should be part of the verification process.

**Comment by nmcharlton** [comment]
This can be a frontend-only piece of work since Treetracker API already supports tree creation (see [API spec](https://raw.githubusercontent.com/Greenstand/treetracker-api/main/docs/api/spec/treetracker.v1.yaml)).
```
POST /api/treetracker/trees
```

**Comment by nmcharlton** [comment]
We could simply replace the _Age_ section in the Verify tags _Approve_ dialog with a checkbox (unchecked by default):
```
[ ] This is a new tree
```

@sebastiangaertner Would that do the trick?
We would still need a way to mark already approved captures as new trees.

@Kpoke Do we record these actions as audit events?

**Comment by Kpoke** [comment]
No, they are not currently recorded as audit events

**Comment by sebastiangaertner** [comment]
@nmcharlton yes this would totally do the trick. In the verification process this needs to be an option and maybe with a bit more UI clarity .

**Comment by dadiorchen** [comment]
I prefer to combine this with the verifying API and use a transaction to do all the work on the cloud site, to achieve data consistency.

**Comment by nmcharlton** [comment]
@dadiorchen @Kpoke What do you think to us adding a flag to the `POST /captures` message to indicate that this is a new tree / first capture?
The transaction would then handle `tree` creation as you suggest.
